<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>priority</class>
 <widget class="QDialog" name="priority">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>367</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>You can rate each item based on your preference</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>universityLocation</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>field</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBox_s">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_s"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_w"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>workplace</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBox_w">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>specialties</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>level</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_f"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_u"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_l"/>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBox_f">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spinBox_u">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinBox_l">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>240</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>use default priorities</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>169</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="save">
      <property name="styleSheet">
       <string notr="true">#save:hover

{
background-color:rgb(0, 103, 159)
}
#save
{
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}</string>
      </property>
      <property name="text">
       <string>save</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="styleSheet">
       <string notr="true">#cancel:hover

{
background-color:rgb(0, 103, 159)
}
#cancel
{
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}</string>
      </property>
      <property name="text">
       <string>cancel</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
